tests/test_data_passable.py

```diff
@@ -34,4 +40,65 @@ def test():
     tuner = irace(scenario, params, target_runner)
     best_conf = tuner.run()
     assert q.get() == 124
-    
\ No newline at end of file
+    
+def test_instances():
+    q = Queue()
+    scenario = dict(
+        instances = [q],
```

Comment:
Why would an instance be a Queue? What is this testing? Could you add a testcase that is more realistic, like instances being a list of functions that need to me minimized?

Reply:
It's because Queue is not serializable and can only work if the queue on the other hand shares the same memory address.

Reply:
But why pass it in "instances" and not via other way? You are using "instances" for something that is not its purpose. 
Does this need to be part of the scenario?

Reply:
I guess it doesn't need to be. But I think it's nice if there are multiple ways to pass stuff around.